src/Map/src/Bridge/Leaflet/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/ux-map-leaflet",
+    "type": "symfony-ux-map-bridge",
+    "description": "Symfony UX Map Leaflet Renderer",
+    "keywords": ["symfony-ux", "map", "leaflet"],
```

Comment:
```suggestion
    "keywords": ["leaflet", "map", "symfony", "ux"],
```

Reply:
"symfony-ux" is a keyword used in every `src/*/composer.json`, at first position :/

Reply:
Yes but this is a bridge :-)